src/org.xtuml.bp.core/models/org.xtuml.bp.core/ooaofooa/Subsystem/Model Class/Model Class.xtuml

```diff
@@ -762,10 +762,8 @@ INSERT INTO O_TFR
 	'',
 	"ba5eda7a-def5-0000-0000-000000000005",
 	1,
-	'  select any ref_obj related by self->PE_PE[R8001]->EP_PKG[R8000]->S_SYS[R1405]->EP_PKG[R1405]->PE_PE[R8000]->O_OBJ[R8001]
-      where selected.Obj_ID == param.ref_obj_id;
-
-    
+	'select any ref_obj from instances of O_OBJ where selected.Obj_ID == param.ref_obj_id;
```

Comment:
[3/9/17, 12:21:20 PM] Bob Mulvey: The change to use “select any FIO” does not seem right to me
[3/9/17, 12:22:13 PM] Bob Mulvey: We need to be looking outside the "current" model , as I recall this is where we use the operation “getVisibleElementsByName”
[3/9/17, 12:22:56 PM] Bob Mulvey: Also, for performance, I would think it would be best to leave that prior query and only do the new if that one is empty